package.json

```diff
@@ -131,8 +131,9 @@
     "screenfull": "^4.0.1",
     "sdp-transform": "^2.14.1",
     "semver": "^7.3.2",
-    "three": "github:mozillareality/three.js#c40ae76f114346b33616f72edf43b4eef26d338d",
+    "three": "github:mozillareality/three.js#0404718c9b3a502ac66408683b2d793a80813a91",
```

Comment:
Note for reviewers:

Range request requires this Three.js PR that has not been merged yet.

https://github.com/mrdoob/three.js/pull/24580

I made a branch `hubs-patches-141-range-requests` from `hubs-patches-141`, did cherry-pick to `hubs-patches-141-range-requests`, and let Hubs Client refer to it.

https://github.com/MozillaReality/three.js/tree/hubs-patches-141
https://github.com/MozillaReality/three.js/tree/hubs-patches-141-range-requests

Another option is applying the change to `hubs-patches-141`.

Reply:
I think we should create a PR against hubs-patches-141, merge it, then do a interactive rebase to make the build. Same as for other patches we have made. Once this lands in upstream and we update three we can remove it.

Reply:
OK, I sent a PR to our Three.js fork.

https://github.com/MozillaReality/three.js/pull/68